src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/ViewHistoryTabAction.kt

```diff
@@ -0,0 +1,28 @@
+package com.atlassian.performance.tools.jiraactions.api.action
+
+import com.atlassian.performance.tools.jiraactions.api.VIEW_HISTORY_TAB
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+
+class ViewHistoryTabAction(
+    private val jira: WebJira,
+    private val meter: ActionMeter,
+    private val issueKeyMemory: IssueKeyMemory
+) : Action {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    override fun run() {
+        val issueKey = issueKeyMemory.recall()
+        if (issueKey == null) {
+            logger.debug("Skipping View History Tab action. I have no knowledge of issue keys.")
+            return
+        }
+
+        meter.measure(key = VIEW_HISTORY_TAB) {
```

Comment:
I think we should add issue information to the measure observation (3rd argument of `meter.measure()`). If I'm correct without it we won't be able to reason based on results if specific issues have this view slower than others.

Reply:
Same for `VIEW_FOCUSED_COMMENT`

Reply:
Good point, I'll add it.